### ISSUE TITLE
Request throws exception. Airtable expects TLS1.2

### ISSUE THREAD

**Issue by brinkdinges** [ngocnicholas/airtable.net#31]
Hi there,

Thanks so much for putting this project together. I added the NuGet package and tried to send my first request, which failed with the following exception:
An error occurred while sending the request. 
Inner exception: The underlying connection was closed
Inner exception: Authentication failed because the remote party has closed the transport stream. 

I cloned the project and it fails at AirtableBase.CreateUpdateReplaceRecord, line 477
var response = await httpClientWithRetries.SendAsync(request);

I was able to fix this by adding TLS1.2 as an option in my project. Airtable stopped supporting 1.0 and 1.1 in July of 2019. My .NET Framework 4.7.2 project apparently does not support this by default, it was set to TLS + Ssl3. I added this:
ServicePointManager.SecurityProtocol = SecurityProtocolType.Tls12 | SecurityProtocolType.Ssl3;

More info:
https://codeshare.co.uk/blog/how-to-fix-the-error-authentication-failed-because-the-remote-party-has-closed-the-transport-stream/
https://support.airtable.com/hc/en-us/articles/360021216813-Ending-support-for-TLS-1-0-and-TLS-1-1

Since I hardly know what I'm doing, I thought I'd let you know :) Maybe you know if it's better to add a warning, add a check or change the default setting for this package.

**Comment by ngocnicholas** [comment]
Airtable's requirement of  TLS1.2 has been supported since Airtable.net 1.0.2 in 2019.
Your issue is similar to the closed one below:
`
https://github.com/ngocnicholas/airtable.net/issues/13
`
I suspect that your real issue is that your framework is not 4.7.2 as you think.

**Comment by brinkdinges** [comment]
Thanks. My project certainly targets 4.7.2, I double-checked, But I run as a plugin in another process, so maybe that process targets an older framework version.

I tried settings ServicePointManager.SecurityProtocol to SecurityProtocolType.SystemDefault, which was added in 4.7, but it does not recognize this value at runtime. So this seems to be the issue indeed. I'll keep adding TLS1.2 manually for now, even though that it advised against.
